gui/control_main.py

```diff
@@ -52,6 +52,7 @@
 from gui.raster import RasterCell, RasterGroup
 from QPeriodicTable import QPeriodicTable
 from threads import RaddoseThread, VideoThread, ServerCheckThread
+from utils import validation
```

Comment:
extracting the validation is a good idea - will be good to remove Albula exceptions on the server side